core/src/main/java/com/homeaway/streamplatform/streamregistry/model/AlertConfig.java

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2018-Present Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamplatform.streamregistry.model;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Builder;
+import lombok.Data;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = AlertConfig.AlertConfigBuilder.class)
+@Builder
+@Data
+public class AlertConfig {
+
+    @Builder
+    public AlertConfig(String type, String destination) {
+        this.type = type;
+        this.destination = destination;
+    }
+
+    /**
+     * Type of alert.
+     */
+    @NotNull
+    String type;
+
+    /**
+     * Additional information to send alert (channel in case of slack, address in case of email, etc.)
+     */
+    @NotNull
+    String destination;
```

Comment:
`@NotEmpty`?

Reply:
1. For email - Where is the email server configured?
2. For slack - Where is the slack token configured?

For both cases, how would credentials/tokens not be exposed?

Reply:
The onus is not on `stream-registry` to actually do anything with these alert configs (at least not in the scope of this PR). For both cases, the specific configurations would be taken care of by components that are going to generate the alerts (i.e. the components which would detect the scenarios in which the alerts are being sent).

Reply:
My point there was - I think you might need additional configurations rather than simply "destination"

Reply:
Makes sense. I can add an additional field for configs.

Reply:
see my slack commentary...